src/scanner/custom-rules/display-accessible-names.ts

```diff
@@ -28,15 +28,18 @@ function hasAccessibleName(node: HTMLElement): boolean {
     // this list of roles and elements were derived from the ARIA 1.2 documentation (Section 5.2.8.6: Roles which cannot be named): https://www.w3.org/TR/wai-aria-1.2/#namefromprohibited
     const nameProhibitedSelectors: string =
         'caption, figcaption, [role=caption], code, [role=code], del, [role=deletion], em, [role=emphasis],[role=generic], ins, [role=insertion], p, [role=paragraph], [role=presentation], [role=strong], strong, sub, sup, [role=subscript], [role=superscript], [role=none]';
-    const nativeElements: string = 'blockquote, figure, form, li, meter, time,section, tr'; // overwriting axe-cores accessible name value as their calculation tends to return the contents of these elements
+    // For certain native elements that do not require attributes, but can only accept names from author-provided attributes as in the "if" check below, axe-core tends to get the accessible names from the elements' contents, which is not entirely right, hence the "if" check to overwrite axe's results
+    const nativeElements: string = 'blockquote, figure, form, li, meter, table, time,section, tr';
     if (axe.utils.matchesSelector(node, nameProhibitedSelectors)) {
         return false;
     }
+    // overwriting axe-cores accessible name value as their calculation tends to return the contents of these elements
     if (
         axe.utils.matchesSelector(node, nativeElements) &&
         node.hasAttribute('aria-label') === false &&
         node.hasAttribute('aria-labelledby') === false &&
-        node.hasAttribute('title') === false
+        node.hasAttribute('title') === false &&
+        node.hasAttribute('summary') === false
```

Comment:
Ideally, this PR would only contain changes related to removing the feature flag. Keeping PRs focused to a single topic improves the code review experience and makes the commit history easier to interpret once the PR has been merged. That said, if you'd still like this change to be a part the PR because it is very small, it would be good to add a note about it in the PR description and even maybe a github comment like this highlighting that the changes were intentional

Reply:
(same applies to all of the changes in this file)